### PR TITLE
changes based on feedback from translation

### PR DESCRIPTION
**Pull request by kalexand-rh**
Aiko had some questions, and I'm rolling some clarifications into the source.

### CODE REVIEW

File: installing/install_config/customizations.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ the `oc explain` command, for example `oc explain builds --api-version=config.op
 All cluster configuration resources are globally scoped (not namespaced) and named `cluster`.
 ////
 Config changes should not require coordinated changes between config resources, if you find
-yourself struggling to update these docs to explain coordinated changes, please reach out 
+yourself struggling to update these docs to explain coordinated changes, please reach out
```
Comment (bobfuru) [inline installing/install_config/customizations.adoc]: I know it's not in the scope of this PR, but this is a run-on sentence that could use a period:
"Config changes should not require coordinated changes between config resources. If you find
yourself struggling..."

File: installing/install_config/customizations.adoc [hunk 2]
```diff
@@ -97,7 +97,7 @@ to control which example imagestreams and templates are installed on the cluster
 == Additional configuration resources
 
 These configuration resources represent a single instance of a particular component, in some cases multiple
-instances can be requested by creating multiple instances of the resource.  In other cases only a specific 
+instances can be requested by creating multiple instances of the resource.  In other cases only a specific
```
Comment (bobfuru) [inline installing/install_config/customizations.adoc]: Also not in scope but could use a period and commas inserted: 
s/component, in some cases multiple/component. In some cases, multiple/
s/In other cases only/In other cases, only

File: networking/enable-cluster-wide-proxy.adoc [hunk 3]
```diff
@@ -14,7 +14,7 @@ The cluster-wide proxy is only supported if you used a user-provisioned infrastr
 
 .Prerequisites
 
-* Review the  xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall[sites that your cluster requires access to] and determine whether any need to bypass the proxy. By default, all cluster egress traffic is proxied, including calls to hosting cloud provider APIs. Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
+* Review the  xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall[sites that your cluster requires access to] and determine whether any need to bypass the proxy. By default, all cluster egress traffic is proxied, including calls to the cloud provider API for the cloud that hosts your cluster. Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
```
Comment (bobfuru) [inline networking/enable-cluster-wide-proxy.adoc]: s/any need to/any of them need to/ (for clarity)

**Review by bobfuru — COMMENTED** [review]
LGTM - added a few minor suggestions for consideration.

File: installing/install_config/customizations.adoc (hunk 1 above)
Comment (kalexand-rh) [reply installing/install_config/customizations.adoc]: It's also in a comment that will never be uncommented. :)

File: networking/enable-cluster-wide-proxy.adoc (hunk 3 above)
Comment (kalexand-rh) [reply networking/enable-cluster-wide-proxy.adoc]: > ! need to v
Avoid using “need to” to refer to a requirement that a product or procedure imposes.

Thanks for pointing that out! I'll pick a different clarification.

**Comment by vikram-redhat** [comment]
@kalexand-rh https://github.com/openshift/openshift-docs/pull/18127

**Pull request merged**